ansible_ai_connect_chatbot/src/AnsibleChatbot/AnsibleChatbot.tsx

```diff
@@ -196,3 +196,4 @@ export const AnsibleChatbot: React.FunctionComponent = () => {
     </>
   );
 };
+AnsibleChatbot.displayName = "AnsibleChatbot";
```

Comment:
At some point we will have to check if this should be named Virtual Assistant?

Reply:
It will be addressed with [AAP-33903](https://issues.redhat.com/browse/AAP-33903). Thanks.